Harvest.Web/Startup.cs

```diff
@@ -209,7 +209,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, AppDbCon
                     name: "default",
                     pattern: "{controller}/{action}/{id?}",
                     defaults: new { controller = "Home", action = "Index" },
-                    constraints: new { controller = "(invoice|rate|permissions)" }
+                    constraints: new { controller = "(invoice|rate|permissions|crop)" }

```

Comment:
This was fun. Without this, I had to put the id as a hidden value on the edit form, otherwise it returned zero in the controller id post.

Reply:
@srkirkland Also, should I remove invoice from here?

Reply:
yea